docs/user-guide/sdks-using.md

```diff
@@ -59,7 +67,7 @@ Define the packages as dependencies in your project and pull them from an online
 
 **Follow these steps:**
 
-1. **(Node SDK only)** Install required dependencies for the Node SDK. In a command-line window, navigate to your project directory and issue the following commands:
+1. **(Node.js SDK only)** Install required dependencies for the Node SDK. In a command-line window, navigate to your project directory and issue the following commands:
```

Comment:
Should step 1 be removed here, or replaced with something more generic? The step to "install required dependencies" seems applicable to both the Node.js and Python SDKs, and is now covered in the Software Requirements section above.